tests/integrationv2/test_ocsp.py

```diff
@@ -119,9 +120,9 @@ def test_s2n_server_ocsp_response(managed_process, cipher, provider, other_provi
         # it immediately after sending the message.
         kill_marker = b"Sent: "
 
-    server = managed_process(S2N, server_options, timeout=2000)
+    server = managed_process(S2N, server_options, timeout=90)
     client = managed_process(provider, client_options,
-                             timeout=2000, kill_marker=kill_marker)
+                             timeout=90, kill_marker=kill_marker)
```

Comment:
Hmmm the original PR had 2000, but no explanation for it and none of the reviewers questioned it. @goatgoose any chance you remember the reasoning from almost 3 years ago? :P

Reply:
I do not remember sadly. I'm guessing it was a mistake though. 2,000 seems way too high to be intentional.